homework4/js/main.js

```diff
@@ -0,0 +1,76 @@
+//Додати 0.1 та 0.2 так, що був правильний математичний результат
+console.log("Sum:", Math.floor((0.1 + 0.2) * 100) / 100);
+
+
+//Авторизація користувача
+
+const userLogin = prompt("Enter your login");
+const userPassword = prompt("Enter your password");
+
+if(userLogin === 'admin' && userPassword === '2pass33210'){
+    console.log("You have successfully logged in!");
+}  else if(userLogin !== 'admin' && userPassword !== '2pass33210'){
+    console.log("Such a user does not have access to the admin panel");
+} else if(userLogin !== 'admin'){
+    console.log("You entered your username or password incorrectly");
+} else if(userPassword !== '2pass33210'){
+    console.log("You entered your username or password incorrectly");
+}
+
+
+//Обрання продуктів користувачем
+
+const userMoney = +prompt("Скільки в вас грошей?");
+
+console.log("Кавун: 21 грн\n Морква: 8 грн\n Картопля: 12 грн\n Яблука: 15 грн");
+
+const userWish = prompt("Що б ви хотіли придбати");
+
+if(userMoney < 8){
+    console.log("На жаль, у Вас недостатньо коштів для купівлі хоча б 1 шт. будь-якого продукту");
+} else if(userWish === 'Кавун' && userMoney < 21){
```

Comment:
You use the names of the products in several places, it would be nice to have these names in constants